### ISSUE TITLE
Nested CEs do not render default content in slots

### ISSUE THREAD

**Issue by quarkus** [webtides/element-js#113]
eg. "nested-shadow-default-tag" actually contains a template which renders something like:
html`<slot>DEFAULT</slot>`;

```
class SlottingParentDefaultTag extends TemplateElement {
	template() {
		return html`<nested-shadow-default-tag></nested-shadow-default-tag> `;
	}
}
```
Expectation: renders "DEFAULT"
Reality: renders ""

**Comment by quarkus** [comment]
i also added one of the tests as a visual test that can be looked at in a browser
<img width="1722" alt="screenshot_mh 2024-03-19 um 22 02 23" src="https://github.com/webtides/element-js/assets/1425838/56a30db3-0c39-410a-bf37-5bf28f0dad76">

**Comment by eddyloewen** [comment]
Has been released with 1.0
